test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/Services/NuGetPackageFileServiceTests.cs

```diff
@@ -280,7 +280,7 @@ public async void AddIconToCache_WhenMissing_IsNotFound()
 
         [Theory]
         [InlineData("icon.png", "icon.png", "icon.png", "")]
-        public async void AddLicenseToCache_WhenAddedTwice_UsesSecond(
+        public async Task AddLicenseToCache_WhenAddedTwice_UsesSecond(
```

Comment:
If it's an async method, it should follow async naming convention:
```suggestion
        public async Task AddLicenseToCache_WhenAddedTwice_UsesSecondAsync(
```